plugins/mumble_plugin_win32.h

```diff
@@ -105,16 +154,25 @@ static bool inline initialize(const std::multimap<std::wstring, unsigned long lo
 	if (!dwPid)
 		return false;
 
+#ifndef USE_PLUGIN_LEGACY_PTR
+	static VOID *pModule;
+#endif
 	pModule=getModuleAddr(modname ? modname : procname);
 	if (!pModule) {
 		dwPid = 0;
 		return false;
 	}
+	pModule32 = reinterpret_cast<uintptr32_t>(pModule);
```

Comment:
I think it might make sense...

But I don't think it would be used in practice...

In practice, the plugins are written to a particular architecture. (i.e., bf4, bf4_x86).

bf4 would use pModule64 (and never touch pModule32). bf4_x86, vice versa.